pkgs/development/libraries/qt-6/srcs.nix

```diff
@@ -1,262 +1,262 @@
+      name = "qtimageformats-everywhere-src-6.3.1.tar.xz";
     };
   };
   qtlanguageserver = {
-    version = "6.3.0";
+    version = "6.3.1";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.3/6.3.0/submodules/qtlanguageserver-everywhere-src-6.3.0.tar.xz";
-      sha256 = "1apfkq5grxkx69d8x7gmj19klr3jypsz1csw6r00q7hf0vvxiakh";
-      name = "qtlanguageserver-everywhere-src-6.3.0.tar.xz";
+      url = "${mirror}/official_releases/qt/6.3/6.3.1/submodules/qtlanguageserver-everywhere-src-6.3.1.tar.xz";
+      sha256 = "1g2azb4mdzh5zp7xc57g8l2a8wfi44wfjm6js88q4mmchyj4f4br";
+      name = "qtlanguageserver-everywhere-src-6.3.1.tar.xz";
     };
   };
   qtlottie = {
-    version = "6.3.0";
+    version = "6.3.1";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.3/6.3.0/submodules/qtlottie-everywhere-src-6.3.0.tar.xz";
-      sha256 = "1svxz5ndljhrn52vyyr1yziar63ksjz78mvaxfhjgdd5pc5mgnrr";
-      name = "qtlottie-everywhere-src-6.3.0.tar.xz";
+      url = "${mirror}/official_releases/qt/6.3/6.3.1/submodules/qtlottie-everywhere-src-6.3.1.tar.xz";
+      sha256 = "1x8wmc6gwmxk92zjcsrbhrbqbfvnk7302ggghld5wk8jk5lsf2vl";
+      name = "qtlottie-everywhere-src-6.3.1.tar.xz";
     };
   };
   qtmultimedia = {
-    version = "6.3.0";
+    version = "6.3.1";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.3/6.3.0/submodules/qtmultimedia-everywhere-src-6.3.0.tar.xz";
-      sha256 = "0gpylyrjkks27y5bfaxqs7idj0wyscpn1kh51i4ahx19z1zj8l6h";
-      name = "qtmultimedia-everywhere-src-6.3.0.tar.xz";
+      url = "${mirror}/official_releases/qt/6.3/6.3.1/submodules/qtmultimedia-everywhere-src-6.3.1.tar.xz";
+      sha256 = "0dkk3lmzi2fs13cnj8q1lpcs6gghj219826gkwnzyd6nmlm280vy";
+      name = "qtmultimedia-everywhere-src-6.3.1.tar.xz";
     };
   };
   qtnetworkauth = {
-    version = "6.3.0";
+    version = "6.3.1";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.3/6.3.0/submodules/qtnetworkauth-everywhere-src-6.3.0.tar.xz";
-      sha256 = "17q6v4d2qglw88gd2i9m4cvvacpfsw6a544g0ch8a0hr56a9hfi0";
-      name = "qtnetworkauth-everywhere-src-6.3.0.tar.xz";
+      url = "${mirror}/official_releases/qt/6.3/6.3.1/submodules/qtnetworkauth-everywhere-src-6.3.1.tar.xz";
+      sha256 = "0apvsb2ip1m3kw8vi9spvf6f6q72ys8vr40rpyysi7shsjwm83yn";
+      name = "qtnetworkauth-everywhere-src-6.3.1.tar.xz";
     };
   };
   qtpositioning = {
-    version = "6.3.0";
+    version = "6.3.1";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/6.3/6.3.0/submodules/qtpositioning-everywhere-src-6.3.0.tar.xz";
-      sha256 = "0vi3123pa9pc4xqh6rgxwz40xvvl4w0x09fn6kdld8s5nbv51vg9";
-      name = "qtpositioning-everywhere-src-6.3.0.tar.xz";
+      url = "${mirror}/official_releases/qt/6.3/6.3.1/submodules/qtpositioning-everywhere-src-6.3.1.tar.xz";
+      sha256 = "0v78wamvdw02kf9rq7m5v24q2g6jmgq4ch0fnfa014p1r978wy06";
+      name = "qtpositioning-everywhere-src-6.3.1.tar.xz";
```

Comment:
> error: derivation A requires non-existent output 'out' from input derivation B

looks like some dependency is not declared
so i have to restart the build multiple times
until by accident, the dependency is in /nix/store

> $ nix-build . -A qt6.qtwebengine
> these 22 derivations will be built:
>   /nix/store/12bws6yn5n4hafkp01jrvlvm6ii35sw4-setup-hook.sh.drv
>   /nix/store/5nk779ghm8dygif8fkyi6ibg88ba2krd-qtserialport-6.3.1.drv
>   /nix/store/x4sawdx3xzwxrf52nx9lwsghfs6dhm10-qtshadertools-6.3.1.drv
>   /nix/store/cidc9vdjgyq2zi5jsjpf29fjp9cgm8ng-qtdeclarative-6.3.1.drv
>   /nix/store/6arridv0q268jrh87ddj5f1nkgn3k0fq-qtpositioning-6.3.1.drv
>   /nix/store/b1yfbdw04nfih0z668jm696glaim78kj-qtwebsockets-6.3.1.drv
>   /nix/store/rx5h1zrdqlhk73j66q3d1y81rds1jn7c-qtwebchannel-6.3.1.drv
>   /nix/store/z7miq2jhp37dcx81mskxh1y88ac016jh-ninja-tokenpool-unstable-2022-02-13.drv
>   /nix/store/w3lfyqq9dp5qxrflsqnvykl1d0f30lx3-qtwebengine-6.3.1.drv
> ...
> error: derivation '/nix/store/6arridv0q268jrh87ddj5f1nkgn3k0fq-qtpositioning-6.3.1.drv' requires non-existent output 'out' from input derivation '/nix/store/5nk779ghm8dygif8fkyi6ibg88ba2krd-qtserialport-6.3.1.drv'

> $ nix-build . -A qt6.qtserialport
> /nix/store/gmbllljkz48lyajgkw2fj6y559svfpw6-qtserialport-6.3.1

> $ nix-build . -A qt6.qtpositioning
> /nix/store/hawd4q0qyp5ggjkq3bij92a2kkfc2n9c-qtpositioning-6.3.1

> $ nix-build . -A qt6.qtwebengine
> these 4 derivations will be built:
>   /nix/store/6arridv0q268jrh87ddj5f1nkgn3k0fq-qtpositioning-6.3.1.drv
>   /nix/store/b1yfbdw04nfih0z668jm696glaim78kj-qtwebsockets-6.3.1.drv
>   /nix/store/rx5h1zrdqlhk73j66q3d1y81rds1jn7c-qtwebchannel-6.3.1.drv
>   /nix/store/w3lfyqq9dp5qxrflsqnvykl1d0f30lx3-qtwebengine-6.3.1.drv
> ...
> error: derivation '/nix/store/w3lfyqq9dp5qxrflsqnvykl1d0f30lx3-qtwebengine-6.3.1.drv' requires non-existent output 'out' from input derivation '/nix/store/6arridv0q268jrh87ddj5f1nkgn3k0fq-qtpositioning-6.3.1.drv'

> $ nix-build . -A qt6.qtwebengine
> this derivation will be built:
>   /nix/store/w3lfyqq9dp5qxrflsqnvykl1d0f30lx3-qtwebengine-6.3.1.drv
> ...

Reply:
> looks like some dependency is not declared
so i have to restart the build multiple times

That sounds like a nix bug, I remember seeing the same symptom in an unrelated issue report: https://github.com/NixOS/nixpkgs/issues/178653

Reply:
ignoring, out of scope